concourse/pipeline.yml

```diff
@@ -351,3 +384,34 @@ jobs:
           CF_SPACE: staging
           CDN_DOMAIN: account.staging.publishing.service.gov.uk
           HOSTNAME: attributes
+  - name: resume-production
+    plan:
+      - get: git-main
+      - task: map-account-url-to-account-app
+        file: git-main/concourse/tasks/map-route.yml
+        params:
+          CF_APP_NAME: govuk-account-manager
+          CF_SPACE: production
+          CDN_DOMAIN: account.publishing.service.gov.uk
+          HOSTNAME: www
+      - task: unmap-account-url-from-error-page
+        file: git-main/concourse/tasks/unmap-route.yml
+        params:
+          CF_APP_NAME: govuk-account-static-errors
+          CF_SPACE: production
+          CDN_DOMAIN: account..publishing.service.gov.uk
+          HOSTNAME: www
+      - task: map-attribute-url-to-attribute-service
+        file: git-main/concourse/tasks/map-route.yml
+        params:
+          CF_APP_NAME: govuk-attribute-service
+          CF_SPACE: production
+          CDN_DOMAIN: account.publishing.service.gov.uk
+          HOSTNAME: attributes
+      - task: unmap-attribute-url-from-error-page
+        file: git-main/concourse/tasks/unmap-route.yml
+        params:
+          CF_APP_NAME: govuk-account-static-errors
+          CF_SPACE: production
+          CDN_DOMAIN: account.publishing.service.gov.uk
```

Comment:
These now mirror what we merged here.

[And the lines above](https://github.com/alphagov/govuk-account-manager-prototype/pull/587/files#diff-7223ad76fd6ece59d0f45aac46c3a233d7f54531b86f43eaac00a466bb4a7561R378). Might just need to remind myself why this shouldn't be the attributes URL 🤔